sql/core/src/test/resources/sql-tests/inputs/charvarchar.sql

```diff
@@ -117,3 +117,10 @@ drop table char_tbl4;
 -- ascii value for Latin-1 Supplement characters
 select ascii('§'), ascii('÷'), ascii('×10');
 select chr(167), chr(247), chr(215);
+
+-- [SPARK-43815] Add to_varchar alias for to_char
+-- TO_VARCHAR()
+-- Alias of TO_CHAR()
```

Comment:
These three lines look repeating. Could you simply the above three lines into a single liner? Maybe, like the following?
```
-- to_varchar is an alias for to_char
```

Reply:
Alright, will do so.